src/discriminator/model_factory.py

```diff
@@ -0,0 +1,118 @@
+from keras.models import Model
+from keras.layers import Input, Conv2D, BatchNormalization, Activation, LeakyReLU, Add, Dense
```

Comment:
BatchNormalization is not used in this code

Reply:
fixed.